internal/backend/local/testing.go

```diff
@@ -32,13 +34,7 @@ func TestLocal(t *testing.T) (*Local, func()) {
 	local.StateWorkspaceDir = filepath.Join(tempDir, "state.tfstate.d")
 	local.ContextOpts = &terraform.ContextOpts{}
 
-	cleanup := func() {
-		if err := os.RemoveAll(tempDir); err != nil {
-			t.Fatal("error cleanup up test:", err)
-		}
-	}
```

Comment:
Huh. How long has this cleanup been unnecessary? I see from looking at the `t.TempDir` source that the directory is removed via `t.Cleanup`, but has that always been true?

Reply:
Yup, `t.TempDir` was added in go1.15 not only to provide temp space for the test, but to automatically clean it up.